src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/model/ContactPurposeType.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.prisonregister.model
+
+import uk.gov.justice.digital.hmpps.prisonregister.exceptions.BadContactPurposeTypeException
+
+enum class ContactPurposeType(val value: String) {
```

Comment:
Is this a good name, any ideas?

Reply:
`Department` perhaps? I think this would be useful for services that might want to contact reception for a variety of reasons